automated-comments/elixir/solution/use_module_doc.md

```diff
@@ -0,0 +1,8 @@
+<!--
+This error is raised when the submission lacks @moduledoc for the Module
+-->
+
+It is recommended to have a [`@moduledoc`](https://hexdocs.pm/elixir/writing-documentation.html#module-attributes) attribute to describe the intent of the Module.
+
+- Documentation is a [first-class citizen](https://hexdocs.pm/elixir/writing-documentation.html) in Elixir -- It is designed to be easy to read and write.
```

Comment:
```suggestion
- Documentation is a [first-class citizen](https://hexdocs.pm/elixir/writing-documentation.html) in Elixir -- Elixir's documentation is designed to be easy to read and write.
```
`It` can refer back to Elixir or first-class citizen, instead of documentation.